tuftool/src/root.rs

```diff
@@ -106,6 +107,9 @@ pub(crate) enum Command {
         ///Optional - Path of older root.json that contains the key-id
         #[structopt(short = "c", long = "cross-sign")]
         cross_sign: Option<PathBuf>,
+        ///Force signing with fewer keys than the specified threshold
+        #[structopt(short = "f")]
+        force: bool,
```

Comment:
I think we should call this something more specific, because "force" could mean overriding any number of checks, and root roles are critical, so it's safer to override specific checks you know are OK to skip.  Something like `--ignore-threshold` or `--allow-keys-under-threshold` perhaps?

Reply:
> I think we should call this something more specific, because "force" could mean overriding any number of checks, and root roles are critical, so it's safer to override specific checks you know are OK to skip. Something like `--ignore-threshold` or `--allow-keys-under-threshold` perhaps?

`--ignore-threshold` works for me (over `--allow-keys-under-threshold`).